proto/prysm/v1alpha1/attestation.go

```diff
@@ -223,15 +252,32 @@ func (a *AttestationElectra) GetCommitteeIndex() (primitives.CommitteeIndex, err
 }
 
 // Version --
-func (a *IndexedAttestation) Version() int {
+func (indexedAtt *IndexedAttestation) Version() int {
```

Comment:
Since you started fixing some receivers, there are a bunch for `*AttestationElectra` that are `att` instead of `a`

Reply:
will double check and fix